core-services/prow/02_config/openshift/machine-api-provider-aws/_prowconfig.yaml

```diff
@@ -0,0 +1,89 @@
+tide:
+  queries:
+  - includedBranches:
+    - community-4.6
+    - community-4.7
+    - openshift-4.1
+    - openshift-4.2
+    - openshift-4.3
+    - openshift-4.4
+    - openshift-4.5
+    - openshift-4.6
+    - openshift-4.7
+    - openshift-4.8
+    - openshift-4.9
+    - release-4.6
+    - release-4.7
+    - release-4.8
+    - release-4.9
```

Comment:
Do we need all the old branches? Or should we just have the latest?

Reply:
I don't know, I left all branches after 4.6 since we support it.

Reply:
But these branches aren't in the new repo are they? Or at least, we won't be building from them?

Reply:
I removed all old branches, let's see what happens.